src/context/AuthPass/index.js

```diff
@@ -123,7 +135,12 @@ class AuthPass extends PureComponent<Props, State> {
         />
       );
     }
-    return children;
+    const contextValue = { skipLockCount, setEnabled };
```

Comment:
I would just pass in setEnabled in the context to avoid rerendering all consumers (because new object ref)

Reply:
RIght, gotcha